konig-core/src/main/java/io/konig/shacl/io/ShapeWriter.java

```diff
@@ -81,7 +82,11 @@ public void writeTurtle(NamespaceManager nsManager, Shape shape, File file) thro
 
 		MemoryGraph graph = new MemoryGraph();
 		graph.setNamespaceManager(nsManager);
-		
+		RdbmsShapeValidator validator = new RdbmsShapeValidator();
```

Comment:
Embedding the RdbmsShapeValidator inside the ShapeWriter is an interesting idea, but it violates the separation of concerns.  The ShapeWriter should be concerned only with writing Shapes -- not editing them.  I'm going to refactor this code and pull out the RdbmsShapeValidator later.  For now, this is fine.